src/components/CreateNimi/themes.ts

```diff
@@ -67,4 +67,10 @@ export const themes: Record<NimiCuratedTheme, ThemeAssets> = {
     logoText: raaveLogoText,
     preview: devconPreview,
   },
+  [NimiThemeType.ETH_PRAGUE_2023]: {
+    type: NimiThemeType.RAAVE,
+    logoImage: raaveLogoImage,
+    logoText: raaveLogoText,
+    preview: devconPreview,
+  },
```

Comment:
yeah we didn't do that part since there werent poaps or we didn't know them....And they won't show since we didn't add any poaps...I will delete both of them from being able to be selected until we have new repo done we can add them there